### PR TITLE
Feature/logstash

### PR DESCRIPTION
**Pull request by PabloPie**
Ready to deploy filebeat on api servers and check logs on logstash server.

### CODE REVIEW

File: roles/apicommon/templates/filebeat.yml.j2 [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+
+  # Optional protocol and basic auth credentials.
+  #protocol: "https"
+  #username: "elastic"
+  #password: "changeme"
+
+#----------------------------- Logstash output --------------------------------
+output.logstash:
+  # The Logstash hosts
+  hosts: [{% for host in groups['logstash']%}"{{ hostvars[host]['backend_ip'] }}:5044",{% endfor %}]
```
Comment (gdchamal) [inline roles/apicommon/templates/filebeat.yml.j2:94]: the final ',' is not a problem ?

File: roles/common/tasks/main.yml [hunk 2]
```diff
@@ -51,3 +51,9 @@
 
 - name: template /etc/hosts
   template: src=hosts.j2 dest=/etc/hosts
+
+- name: copy syslog file
+  copy:
+    src: rsyslog.conf
+    dest: /etc/rsyslog.conf
+    mode: 0644
```
Comment (gdchamal) [inline roles/common/tasks/main.yml:59]: we must notify a restart to rsyslog (add a handler for such restart action)

File: roles/apicommon/templates/filebeat.yml.j2 (hunk 1 above)
Comment (PabloPie) [reply roles/apicommon/templates/filebeat.yml.j2:94]: Not for Filebeat apparently.

File: roles/logstash/templates/jvm.options [hunk 3]
```diff
@@ -0,0 +1,80 @@
+## JVM configuration
```
Comment (gdchamal) [inline roles/logstash/templates/jvm.options:1]: I don't see installation of this file on logstash host

**Review by gdchamal — APPROVED** [review]

**Pull request merged**